zoola-academy-homework.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.iml files should be removed from the PR

Reply:
Removed.

Reply:
still there

Reply:
![Screenshot 2022-08-22 at 16 00 07](https://user-images.githubusercontent.com/108929952/185927307-1337ff79-850d-4df8-8b26-9cc94f002129.png)

Reply:
Just deleted this file. Hope this doesn't ruin anything ;)

Reply:
iml files are just used by Intellij, and it should be able to recreate it :)

Unfortunately, the file is still there - check the list of modified files for this MR - https://github.com/zoolacode/zoola-academy-homework/pull/116/files

Reply:
Also, with this commit you've pushed all compiled files for the work (they have a file extension .class). Please remove them as well from the PR

Reply:
Taras, thank you for your patience, it seems to me I've figured this out. I have added the out folder and *.iml to the global .gitignore file in the home directory.